### ISSUE TITLE
Issues with syncing neo4j property arrays to Elasticsearch

### ISSUE THREAD

**Issue by foobargeez** [graphaware/neo4j-to-elasticsearch#20]
Hello,
I am using v2.3.3 connector.  I created a simple node by running:

`MERGE (n:Artist {name:"hehe"}) ON MATCH SET n._otherNames=["\"another name\""]`

I have the following in Elasticsearch:
```
{
...
  },
  "hits" : {
    "total" : 1,
    "max_score" : 4.9270635,
    "hits" : [ {
      "_index" : "neo4j-index",
      "_type" : "Artist",
      "_id" : "2f4eb740-4deb-11e6-b2cf-5254000cab31",
      "_score" : 4.9270635,
      "_source" : {
        "name" : "hehe",
        "uuid" : "2f4eb740-4deb-11e6-b2cf-5254000cab31",
        "_otherNames" : "[Ljava.lang.String;@23972ed6"
      }
    } ]
  }
}
```

The `_otherNames` property in Elasticsearch document has an unexpected value.  Am I hitting a bug somewhere?

Thanks in advance!

**Comment by alenegro81** [comment]
You are creating a property that is an array. Arrays are still not supported, but this is an interesting extension. We will work on this

**Comment by davidrapin** [comment]
Good point, we should update the default mapping to handle this case smoothly.

**Comment by ikwattro** [comment]
Solved by #21

**Comment by foobargeez** [comment]
Thanks for the fast response!  Looking forward for the release!
